### DISCUSSION TITLE
Backup's are failing with error creating volume snapshot: admission webhook error

### DISCUSSION THREAD

**Discussion by gundakishore15** [vmware-tanzu/velero#7401] [Community support Q&A]
Backup's are failing with below error.  Could you please help me on this
rpc error: code = Unknown desc = error creating volume snapshot: admission webhook \"validation-webhook.snapshot.storage.k8s.io\" denied the request: expect resource to be {snapshot.storage.k8s.io v1 volumesnapshots}, {snapshot.storage.k8s.io v1 volumesnapshotcontents}, or {snapshot.storage.k8s.io v1 volumesnapshotclasses}, but found {snapshot.storage.k8s.io v1beta1 volumesnapshots}" logSource="pkg/backup/backup.go:417"

**Comment by smoshiur1237** [comment]
Hi @gundakishore15 , I am seeing the same error with `kubernetes-csi/external-snapshotter` . Is there any solution or workaround that you have used to overcome this error?
